### PR TITLE
docs(vrl): fix vrl code example

### PR DESCRIPTION
**Pull request by alexandre1984rj**
This update addresses the incorrect usage of the `to_timestamp` function in the provided VRL code example. To ensure proper parsing of the timestamp format, I have included the `parse_timestamp` function, based on a more accurate example that effectively handles timestamp conversion. The original code attempted to convert a timestamp using a non-existent function, which could lead to confusion for users.

Playground Link: https://playground.vrl.dev/?state=eyJwcm9ncmFtIjoiLiA9IHBhcnNlX2pzb24hKHN0cmluZyEoLm1lc3NhZ2UpKVxuXG4ucGFyc2VkX3RpbWVzdGFtcCA9IHBhcnNlX3RpbWVzdGFtcCEoLnRpbWVzdGFtcCwgZm9ybWF0OiBcIiVZLSVtLSVkVCVIOiVNOiVTLiVmWlwiKVxuXG5pZiBpc190aW1lc3RhbXAoLnBhcnNlZF90aW1lc3RhbXApIHtcbiAgLnRpbWVzdGFtcCA9IHRvX3VuaXhfdGltZXN0YW1wKC5wYXJzZWRfdGltZXN0YW1wKVxufSBlbHNlIHtcbiAgLnRpbWVzdGFtcCA9IHRvX3VuaXhfdGltZXN0YW1wKG5vdygpKSAgXG59XG5cbmRlbCgudXNlcm5hbWUpXG5kZWwoLnBhcnNlZF90aW1lc3RhbXApXG4ubWVzc2FnZSA9IGRvd25jYXNlKHN0cmluZyEoLm1lc3NhZ2UpKSIsImV2ZW50Ijp7Im1lc3NhZ2UiOiJ7XCJzdGF0dXNcIjoyMDAsXCJ0aW1lc3RhbXBcIjpcIjIwMjEtMDMtMDFUMTk6MTk6MjQuNjQ2MTcwWlwiLFwibWVzc2FnZVwiOlwiU1VDQ0VTU1wiLFwidXNlcm5hbWVcIjpcInViNDBmYW40bGlmZVwifSJ9LCJpc19qc29ubCI6ZmFsc2UsImVycm9yIjpudWxsfQ%3D%3D

image

### CODE REVIEW

**Review by pront — APPROVED** [review]
Thank you @alexandre1984rj!

FYI: You can use the share button and share the playground link.

File: website/content/en/docs/reference/vrl/_index.md [hunk 1]
```diff
@@ -45,22 +45,35 @@ Let's have a look at a more complex example. Imagine that you're working with
...
+{
+  "message": "{\"status\":200,\"timestamp\":\"2021-03-01T19:19:24.646170Z\",\"message\":\"SUCCESS\",\"username\":\"ub40fan4life\"}"
+}
 ```
 
 You want to apply these changes to each event:
 
 - Parse the raw string into JSON
-- Reformat the `time` into a UNIX timestamp
+- Attempts to convert the timestamp and checks if the conversion was successful
```
Comment (drichards-87) [inline website/content/en/docs/reference/vrl/_index.md:56]:
```suggestion
- Attempt to convert the timestamp and checks if the conversion was successful
```

Comment (drichards-87) [reply website/content/en/docs/reference/vrl/_index.md:56]: This suggestion only applies if the user is the one doing this. If it's the program, then "attempts" makes sense. However, I would suggest adding that to the beginning of the sentence, e.g., "The program attempts" or "Vector attempts", whichever is more accurate.

**Review by drichards-87 — APPROVED** [review]
Left a small suggestion from Docs and approved the PR.

**Pull request merged**